spatial/uptime_worker_test.json

```diff
@@ -0,0 +1,121 @@
+{
+	"template": "n1standard4_std40_action1g1",
+	"world": {
+		"dimensions": {
+			"x_meters": 10000,
+			"z_meters": 10000
+		},
+		"chunk_edge_length_meters": 50,
+		"legacy_flags": [
+			{
+				"name": "bridge_qos_max_timeout",
+				"value": "0"
+			},
+			{
+				"name": "bridge_soft_handover_enabled",
+				"value": "false"
+			},
+			{
+				"name": "worker_start_connection_timeout_ms",
+				"value": "360000"
+			},
+			{
+				"name": "enable_chunk_interest",
+				"value": "false"
+			}
+		],
+		"snapshots": {
+			"snapshot_write_period_seconds": 0
+		}
+	},
+	"load_balancing": {
+		"layer_configurations": [
+			{
+				"layer": "UnrealWorker",
+				"rectangle_grid": {
+					"cols": 3,
+					"rows": 3
+				},
```

Comment:
What is the point of this document's existence? If we want to customize the number of rows and columns, will the data configured here have an impact?

Reply:
This document is meant to remind that we have new worker flags, and our new logic is to determine those values through worker flag instead of code. For the rows and cols, they are used when load balancing to divide the world into grids.

Reply:
This is a sample of how to using the added worker flags. I think it would be better if we have some comments in the header file, like how to use this feature.